app/models/vm_or_template/right_sizing.rb

```diff
@@ -163,16 +163,8 @@ def max_mem_usage_absolute_average_max_over_time_period
   end
 
   def cpu_usagemhz_rate_average_max_over_time_period
-    if has_attribute?("cpu_usagemhz_rate_average_max_over_time_period")
-      self["cpu_usagemhz_rate_average_max_over_time_period"]
-    else
-      opts = {
-        :end_date => Time.now.utc,
-        :days     => Metric::LongTermAverages::AVG_DAYS
-      }
-      VimPerformanceAnalysis.find_perf_for_time_period(self, "daily", opts)
-                            .maximum(:cpu_usagemhz_rate_average)
-    end
+    perfs = VimPerformanceAnalysis.find_perf_for_time_period(self, "daily", :end_date => Time.now.utc, :days => Metric::LongTermAverages::AVG_DAYS)
+    perfs.collect(&:abs_max_cpu_usagemhz_rate_average_value).compact.max
   end
 
   def derived_memory_used_max_over_time_period
```

Comment:
is the same problem here?

Reply:
Yes, a similar problem. I am yet to find the source of this metric.

Reply:
ok, BZ mentions this, but it's ok to fix the memory as another PR